README.md

```diff
@@ -1050,6 +1050,15 @@ Kind: *Indication*
 Errors: 
 - `BadOrigin`
 
+### `SetAssetClaimer`
+
+Set a claimer to the dropped assets.
+
+The dropped assets are the assets that are still located in the `Holding Register` after a program finishes its execution.
+
+Kind: *Instruction*
```

Comment:
missing `Operands` section with the `MultiLocation`

Reply:
Fixed